common/display/src/main/scala/uk/ac/wellcome/display/models/DisplayAccessCondition.scala

```diff
@@ -0,0 +1,53 @@
+package uk.ac.wellcome.display.models
+
+import java.time.LocalDate
+import io.circe.generic.extras.JsonKey
+import io.swagger.v3.oas.annotations.media.Schema
+import uk.ac.wellcome.models.work.internal.{AccessCondition, AccessStatus}
+
+@Schema(
+  name = "AccessCondition"
+)
+case class DisplayAccessCondition(
+  status: DisplayAccessStatus,
+  terms: Option[String],
+  to: Option[LocalDate],
+  @JsonKey("type") @Schema(name = "type") ontologyType: String =
+    "AccessCondition"
+)
+
+object DisplayAccessCondition {
+
+  def apply(accessCondition: AccessCondition): DisplayAccessCondition =
+    DisplayAccessCondition(
+      status = DisplayAccessStatus(accessCondition.status),
+      terms = accessCondition.terms,
+      to = accessCondition.to,
+    )
+}
+
+@Schema(
+  name = "AccessStatus"
+)
+case class DisplayAccessStatus(
```

Comment:
minor: should this be in a separate file?

Reply:
We are not adhering to class per file or anything elsewhere, so think it makes sense to have this here as it is just a sub field within `DisplayAccessCondition`